manifests/consul_exporter.pp

```diff
@@ -63,6 +65,7 @@
   String $log_level,
   String $package_ensure,
   String $package_name,
+  String $service_name,
```

Comment:
for all `$service_name` parameters please use `String[1]` to prohibit empty empty strings

Reply:
Sure, will adapt the PR. Shall I change it for `$package_name` too?

As far as I remember I always have just copied the package-name-line as base and adapted it for the service. So there might be a mixture of `String` and `String[1]` there too.

Reply:
yes please :)